packages/neo-one-node-http-rpc/src/rpcServer$.ts

```diff
@@ -70,6 +87,9 @@ export const rpcServer$ = ({
       const rpcMiddleware = rpc({ blockchain, node });
       router.use(context({ monitor }));
 
+      router.use(rateLimit(rateLimitOptions));
```

Comment:
There should be a way to disable both of these. Add an optional `disabled` to each of the options and check if it's present.